packages/app/src/app/pages/Sandbox/Editor/index.tsx

```diff
@@ -177,6 +178,46 @@ const Editor = () => {
                         }
                   }
                 />
+                {showNewSandboxModal ? (
+                  <Element
+                    css={css({
+                      width: '100vw',
+                      height: '100vh',
+                      overflow: 'hidden',
+                      position: 'fixed',
+                      zIndex: 10,
+                      top: 0,
+                      left: 0,
+                    })}
+                  >
+                    <Element
+                      css={css({
+                        background: 'rgba(0,0,0,0.4)',
+                        width: '100vw',
+                        height: '100vh',
+                        position: 'fixed',
+                      })}
+                    />
+                    <Element margin={6}>
+                      <Element marginTop={80}>
+                        <Stack align="center" justify="center">
+                          <Element
+                            css={css({
+                              backgroundColor: 'sideBar.background',
+                              maxWidth: '100%',
+                              width: 1200,
+                              position: 'relative',
+                              zIndex: 100,
+                            })}
+                            marginTop={8}
+                          >
+                            <CreateSandbox />
```

Comment:
One feature request from me: I'd love to start using this in more places, but nowadays I often need to specify where the sandbox will be created in the dashboard by passing the `collectionId` prop. Would it be possible to allow setting this in the URL (something like `/s/?collectionId=blaaah`) and that it can be passed as a prop in here?